hms-app/src/common/types.tsx

```diff
@@ -42,27 +42,22 @@ export type CategoryPreview = {
 }
 
 export type Category = {
-  errorCategoryData: boolean
-  isLoadingCategoryData: boolean
   id: string
   title: string
   description: string
   hackathonId: string
 }
 
-export type SkillPreview = {
-  id: string
-  name: string
-}
-
-export type OwnerPreview = {
+export type ParticipantPreview = {
   id: string
   user: User
 }
 
-export type ParticipantPreview = {
+export type Participant = {
+  userId: string
+  hackathonId: string
   id: string
-  user: User
+  creationDate: string
```

Comment:
this could be a Date object, you can pass a string into `new Date()` and so long as we don't return bad stuff from the backend it should always work